gensim/models/poincare.py

```diff
@@ -514,6 +516,8 @@ def train(self, epochs, batch_size=10, print_every=1000, check_gradients_every=N
         """
         if self.workers > 1:
             raise NotImplementedError("Multi-threaded version not implemented yet")
+        # Some divide-by-zero results are handled explicitly
+        old_settings = np.seterr(divide='ignore', invalid='ignore')
```

Comment:
Why? You mean that division by zero is expected and you process this situation in code, I'm correct?

Reply:
Yeah, it happens in `PoincareBatch`. I'm setting it here to avoid repeated calls to `np.seterr`